dashbuilder-client/dashbuilder-navigation-client/src/main/java/org/dashbuilder/client/navigation/widget/NavCarouselWidgetView.java

```diff
@@ -93,10 +86,10 @@ public void errorNavItemsEmpty() {
     }
 
     @Override
-    public void deadlockError() {
+    public void errorDeadlock(String explanation) {
         Div div = (Div) Window.getDocument().createElement("div");
         div.setClassName(slidesDiv.getChildNodes().getLength() == 0 ? "item active" : "item");
-        alertBox.setMessage(NavigationConstants.INSTANCE.navCarouselDragComponentDeadlockError());
+        alertBox.setMessage(NavigationConstants.INSTANCE.navCarouselDragComponentDeadlockError() + " " + explanation);
```

Comment:
maybe use uberfire standart alert panels.

Reply:
Do you have a link to an example?